src/stores/configStore.js

```diff
@@ -130,25 +141,69 @@ export class ConfigStore {
       }
     });
 
-    // Set default selected locations
-    this.selectedLocationNodes = [
-      getLocationByNameAndLevel(
-        this.locationDataStoreInstance.selectTree,
-        'USA',
-        'country',
-        true
-      )[0],
-      getLocationByNameAndLevel(
-        this.locationDataStoreInstance.selectTree,
-        'Canada',
-        'country',
-        true
-      )[0],
-    ].filter((node) => node !== undefined);
-    initialConfigValues['selectedLocationNodes'] = this.selectedLocationNodes;
-    this.initialConfigValues[
-      'selectedLocationNodes'
-    ] = this.selectedLocationNodes.slice();
+    this.urlParams = parseQueryParams();
```

Comment:
Why not use `URLSearchParams()` (https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) like you did before?